docs/unified/hardkernel/building-manual.md

```diff
@@ -0,0 +1,46 @@
+# Dasharo firmware building guide
+
+## Intro
+
+This guide shows how to build Dasharo firmware for Hardkernel devices.
+
+## Requirements
+
+- Docker
+    + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+- Git
+    + `sudo apt-get install git`
+
+## Building
+
+To build Dasharo firmware image, first clone the coreboot repository:
+    ```bash
+    git clone https://github.com/Dasharo/coreboot.git
+    ```
+then follow the steps below:
+
+=== "Odroid H4+"
+
+    1. Checkout to the device's branch:
+            ```bash
+            cd coreboot
+            git checkout hardkernel_odroid_h4/release
```

Comment:
We do not use branch per platform any longer

Reply:
Shouldn't we update other platform docs if we don't do it like that anymore?

Is that change ok?
https://github.com/Dasharo/docs/pull/903/commits/07655bc698e7aee93f2dd14e44b96ea6d1bee70f#diff-a31f78a5db1f125893b99789f2b960388646cb819668145d0bc392617ea376e6R25-R36

Reply:
Old platforms may still have old releases that build from those branches. We cannot remove and break those.